src/main/java/com/exactpro/th2/common/schema/filter/strategy/impl/AbstractFilterStrategy.java

```diff
@@ -62,22 +63,24 @@ private boolean checkValues(Map<String, String> messageFields, MultiValuedMap<St
         });
     }
 
-    private boolean checkValue(String value1, FieldFilterConfiguration filterConfiguration) {
-        if (StringUtils.isEmpty(value1)) {
+    private boolean checkValue(String value, FieldFilterConfiguration filterConfiguration) {
+        if (StringUtils.isEmpty(value)) {
             return false;
         }
 
-        var value2 = filterConfiguration.getExpectedValue();
+        var valueInConf = filterConfiguration.getExpectedValue();
 
         switch (filterConfiguration.getOperation()) {
```

Comment:
Please, leave a comment about migrating to switch-expression when migrating to a newer java